linode/networkingip/framework_resource_schema.go

```diff
@@ -26,6 +28,15 @@ var frameworkResourceSchema = schema.Schema{
 				int64planmodifier.UseStateForUnknown(),
 			},
 		},
+		"rdns": schema.StringAttribute{
+			Description: "The reverse DNS assigned to this address. " +
+				"For public IPv4 addresses, this will be set to " +
+				"a default value provided by Linode.",
+			Computed: true,
```

Comment:
I originally wanted to support configuring the RDNS from this field but it ended up bloating the resource logic and wouldn't be a common use case vs the dedicated RDNS resource.